createdisk.sh

```diff
@@ -16,7 +16,7 @@ BASE_OS=fedora-coreos
 
 INSTALL_DIR=${1:-crc-tmp-install-data}
 
-VM_IP=$(sudo virsh domifaddr ${CRC_VM_NAME} | grep vnet0 | awk '{print $4}' | sed 's;/24;;')
+VM_IP=$(sudo virsh domifaddr ${CRC_VM_NAME} | grep vnet | awk '{print $4}' | sed 's;/24;;')
```

Comment:
to match 'any' vnet device or is this the full device name?

Reply:
any vnet device, on one of my machines 
```
$ sudo virsh domifaddr crc-podman
 Name       MAC address          Protocol     Address
-------------------------------------------------------------------------------
 vnet42     52:54:00:fd:49:44    ipv4         192.168.122.37/24
```